### PR TITLE
Sidebar

### PR DESCRIPTION
**Pull request by Mounir-kh**

### CODE REVIEW

**Review by OmarRamoun — CHANGES_REQUESTED** [review]
# 🟠 Project Invalid! ⚓

## Hi <STUDENT_MENTION> 👋

Unfortunately 😕, the project is *invalid* (that means I can not review it because you missed some rules that I have listed for you below ⬇️).

While you made a **really great** effort 💪 in this project that I really appreciate 🙇, unfortunately, _I cannot proceed to review your code_.

## Invalid Code Review Request

You have submitted a project, that has no PR description

Your Code Review Request will be marked as **invalid** in your Dashboard. **Don't worry** though, You still have **3** More attepmts. So please submit a new one once you are ready 🙏.

> **Note:** In case you still have any questions or don't understand what is going on, please contact me at [Ramoun](#thank-you-for-your-time-🙏-and-call-me-if-you-need-help-or-clarification-☎️).
> **Note:** You can submit a new project by clicking on the **Submit Project** button in the **Dashboard**.

Cheers and Happy coding!👏👏👏

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. So, you can skip them if you don't have the time but remember to come back and implement them because they are going to enhance the quality of your code_

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@mrramoun](https://github.com/mrramoun)
- This is My **Twitter**: [@ramoun16](https://twitter.com/ramoun16)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="app-wrap">
+    <header class="header1">
+        <img src="Images/My Logo.png">
```
Comment (roshan-bajgain) [inline index.html:13]: - [ ] Kindly wrap the logo with `a` tag because it is a link.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        <img src="Images/menu.png">
+    </header>
```
Comment (roshan-bajgain) [inline index.html:15]: - [ ] The hamburger button on the right should have a `nav` tag because for better semantic HTML.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+<header class="content">
+    <div class="header2">
+        <h1>I'm Mounir Glad to see you ! <br>I'm a software developer </h1>
```
Comment (roshan-bajgain) [inline index.html:18]: - [ ] Kindly change this `header tag` with `section tag` for better semantic HTML

File: index.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        <a href="#">
+            <img src="Images/medium.png" >
+        </a>
+        <a href="#">
+        <img src="Images/github.png" >
+        </a>
+        <a href="#">
+            <img src="Images/angellist.png" >
+        </a>
+    </div>
```
Comment (roshan-bajgain) [inline index.html:46]: - [ ] The list of icons should be a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Kindly fix it

File: index.html [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+            <img src="Images/twitter.png" >
+        </a>
+        <a href="#">
+         <img src="Images/Linkedin icon.png" >
+        </a>
+        <a href="#">
+            <img src="Images/medium.png" >
+        </a>
+        <a href="#">
+        <img src="Images/github.png" >
```
Comment (roshan-bajgain) [inline index.html:41]: - [ ] Kindly stick with the Figma design as much as possible (color, font family, size, and so on).  your design is totally different from Figma
### Your design
yours

### Figma design
Figma-design

**Review by roshan-bajgain — COMMENTED** [review]
Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight :tada: :champagne: 
- Nice README file :heavy_check_mark:

## Required Changes ♻️
- [ ] Kindly update descriptive pull request and never merge the pull request without approval use the same pull request for another reviewer after making changes. 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by koomekevo — COMMENTED** [review]
Hi @Mounir-kh, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly adjust your design as the previous reviewer suggested in accordance with the Figma design. Below is your current design. 

mounirwrong

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**